infra-templates/ipsec/12/rancher-compose.yml

```diff
@@ -57,3 +57,15 @@
       type: "string"
       default: '1024'
       required: true
+    - variable: "IPSEC_IKE_SA_REKEY_INTERVAL"
+      label: "IPSec IKE_SA rekey interval"
+      description: "Time interval for IPSec strongswan configuration to rekey IKA_SAs"
+      type: "string"
+      default: '4h'
```

Comment:
if the values are `1h`, what are other possible values? this seems like it will be easy to put in a wrong value.